README.md

```diff
@@ -21,5 +21,147 @@ fault-tolerant workflows suited for preservation.
 
 Visit https://enduroproject.netlify.com for more details.
 
-[National Health Archive (NHA)]: https://www.piql.com/norwegians-digital-health-data-to-be-preserved-for-future-generations/
-[National Center for Truth and Reconciliation (NCTR)]: https://nctr.ca/about/about-the-nctr/our-mandate/
+## Local environment
+
+Enduro uses Tilt to set up a local environment building the Docker images in a
+Kubernetes cluster. It has been tested with k3d, Minikube and Kind.
+
+### Requirements
+
+- [Docker] (v18.09+)
+- [kubectl]
+- [Tilt] (v0.22.2+)
+
+A local Kubernetes cluster:
+
+- [k3d] _(recomended, used in CI)_
+- [Minikube] _(tested)_
+- [Kind] _(tested)_
+
+It can run with other solutions like Microk8s or Docker for Desktop/Mac and
+even against remote clusters, check Tilt's [Choosing a Local Dev Cluster] and
+[Install] documentation for more information to install these requirements.
+
+Additionally, follow the [Manage Docker as a non-root user] post-install guide
+so that you don’t have to run Tilt with `sudo`.
+
+### Set up
+
+Start a local Kubernetes cluster with a local registry. For example, with k3d:
+
+```
+k3d cluster create sdps-local --registry-create sdps-registry
+```
+
+Make sure kubectl is available and configured to use that cluster:
+
+```
+kubectl config view
+```
+
+Clone this repository and move into its folder:
+
+```
+git clone https://github.com/artefactual-sdps/enduro.git
+cd enduro
+```
+
+Bring up the environment:
+
+```
+tilt up
+```
+
+While the Docker images are built/downloaded and the Kubernetes resources are
+created, hit `space` to open the Tilt UI in your browser. Check the [Tilt UI]
+documentation to know more about it.
+
+### Access
+
+There are four services available from the host:
+
+- Enduro dashboard: http://localhost:3000
+- Minio console: http://localhost:7460 (minio/minio123)
+- Temporal UI: http://localhost:7440
+- Opensearch dashboards: http://localhost:7500
+
+### Live updates
+
+Tilt will watch for file changes in the project folder and it will sync those
+changes, rebuild the Docker images and recreate the resources when necessary.
+The `enduro-dashboard` uses Vite to serve the application in development with
+hot reload, while the Go services require to recreate the images to build the
+binaries and they take longer to update.
```

Comment:
I think you want to explain that the dashboard hot reload is fast and the Go rebuilds are not, but I find this a bit difficult to parse/understand.